Cargo.toml

```
@@ -87,8 +87,8 @@ fuel-core-xtask = { version = "0.0.0", path = "./xtask" }
 fuel-gas-price-algorithm = { version = "0.36.0", path = "crates/fuel-gas-price-algorithm" }
 
 # Fuel dependencies
-fuel-vm-private = { version = "0.57.0", package = "fuel-vm", default-features = false }
-
+fuel-vm-private = { git = "https://github.com/FuelLabs/fuel-vm.git", branch = "feature/predicate-ldc", package = "fuel-vm", default-features = false }
+#fuel-vm-private = { path = "../fuel-vm/fuel-vm", package = "fuel-vm", default-features = false }
```

why two definitions?
```suggestion
fuel-vm-private = { git = "https://github.com/FuelLabs/fuel-vm.git", branch = "feature/predicate-ldc", package = "fuel-vm", default-features = false }
```